types/jquery/v2/index.d.ts

```diff
@@ -3637,6 +3637,13 @@ interface JQuery {
      */
     not(selector: string): JQuery;
     /**
+     * Remove elements from the set of matched elements.
+     *
+     * @param selector An array of elements to match against the set.
+     * @see {@link https://api.jquery.com/not/#not-selector}
+     */
+     not(selector: any[]): JQuery;
+     /**
```

Comment:
Could you merge the two definitions into one?

```suggestion
    not(selector: string | any[]): JQuery;
     /**
```

Reply:
@martin-badin sure, good idea - done.